### DISCUSSION TITLE
自动分桶数推算功能貌似没有实现？

### DISCUSSION THREAD

**Discussion by DA1OOO** [apache/doris#27910] [A - General / Q&A] [answered]
建表使用的`BUCKETS AUTO` ，但是建表的时候没有指定参数`"estimate_partition_size" = "100G"` 
![image](https://github.com/apache/doris/assets/56857867/5cb1cee3-ba68-4fc7-bf02-516d3fe80ca8)

我看文档写的默认没有指定的话，初次分桶大小按10G来算分桶数，我的第一个分区buckets数确实为10
![image](https://github.com/apache/doris/assets/56857867/7f6912ca-3b97-4fd2-a720-5706b3d047a1)
但是为什么后面的分区buckets全部为10，官方文档不是写着会根据前七天的数据量来推算bucket数吗，我每个分区5G的数据应该分桶数大概是5。
![image](https://github.com/apache/doris/assets/56857867/aef4a16b-3815-4ee9-8d6d-0ced67c6a33e)
我最新的分区数的bucket还是10。
我看了一些小表bucket还是一直保持在10。
第一个分区：
![image](https://github.com/apache/doris/assets/56857867/334d750e-66c0-4690-ba67-d5ba9ac2e1af)
最新的分：
![image](https://github.com/apache/doris/assets/56857867/47cd8300-326d-46b6-b36e-05f831b7c4cc)

**Comment by DA1OOO** [comment]
是漏了什么配置吗

**Reply by JackDrogon** [reply]
建表语句是什么呢

**Comment by DA1OOO** [comment]
新增分区方式：
ALTER TABLE example_db.my_table
ADD PARTITION p1 VALUES [("2014-01-01"), ("2014-02-01")); 
没有指定分桶信息。

**Reply by JackDrogon** [reply, marked as answer]
手动新增是不会使用重新计算的，只有dynamic partition中bucket auto按照历史数据评估才会生效

**Comment by ynzzxc** [comment]
我也遇到了类似的问题，我尝试了各种配置，自动分桶 buckets 始终为10，请问你解决了这个问题吗

**Reply by DA1OOO** [reply]
没有，只有初始的时候填一下预估size，才能改掉buckets数量，但是好像不会根据数据的变化动态推算bucket数。
